source/puppet/4.4/reference/install_osx.md

```diff
@@ -29,94 +38,119 @@ To install on other operating systems, see the pages linked in the navigation si
+
+    sudo puppet resource service <SERVICE NAME> ensure=stopped enable=false
 
 ### Installing with the GUI
 
-Double-click the `puppet-agent` disk image you downloaded. This mounts it at `/Volumes/<DMG NAME>`.
+To install Puppet with a graphic user interface:
+
+1.  Double-click the `puppet-agent` disk image you downloaded.
 
-A Finder window appears showing the disk's contents: a single `puppet-agent-<VERSION>-installer.pkg` file. Double-click the package file, and follow the installer prompts to install it. When installation finishes, Puppet agent and MCollective will be running.
+    OS X verifies the image, and then mounts it at `/Volumes/<DMG NAME>`. A Finder window then appears showing the image's contents: a single `puppet-agent-<PACKAGE VERSION>-<RELEASE NUMBER>-installer.pkg` file.
 
-After installing, unmount and delete the disk image.
+2.  Double-click the package file, and then follow the installer prompts.
+
+    Once the installation is complete, the `puppet` and `mcollective` services launch automatically.
+
+3.  Unmount the disk image. You can then delete the image.
 
 ### Installing on the command line
 
-Alternately, you can use the `hdiutil` and `installer` commands to mount the disk image and install the package from the command line.
+You can use the `hdiutil` and `installer` commands to mount the disk image, and then install the package from the command line.
 
-First, mount the disk image with:
+1.  Mount the disk image by running:
 
-    sudo hdiutil mount <DMG FILE>
+        sudo hdiutil mount <DMG FILE>
 
-Next, locate the `.pkg` file in the mounted volume and install it with:
+2.  Locate the `.pkg` file in the mounted volume and install it by running:
 
-    sudo installer -pkg /Volumes/<IMAGE>/<PKG FILE> -target /
+        sudo installer -pkg /Volumes/<IMAGE>/<PKG FILE> -target /
 
-When installation finishes, Puppet agent and MCollective will be running.
+        Once the installation is complete, the `puppet` and `mcollective` services launch automatically.
 
-After installing, unmount the disk image with:
+3.  Unmount the disk image by running:
 
-    sudo hdiutil unmount /Volumes/<IMAGE>
+        sudo hdiutil unmount /Volumes/<IMAGE>
 
-You can then delete the `.dmg` file.
+    You can then delete the disk image.
 
 ### Upgrading with Puppet
 
-Puppet includes a `package` resource provider for OS X that can install `.pkg` files from a disk image. If you already have Puppet installed, you can use the `puppet resource` command to upgrade with fewer steps.
+Puppet includes a [`pkgdmg` package provider](./type.html#package-provider-pkgdmg) that can install OS X installation packages (`.pkg` files) from a disk image (`.dmg` file). If you already have Puppet installed, you can use the `puppet resource` command to upgrade with fewer steps.
```

Comment:
I'd be tempted to drop that whole sentence. At this point, I don't think the reader needs to know what providers do what. (And it doesn't seem like the other `puppet resource` examples in this set of pages go into which provider will be doing the work for them.